src/sentry/api/endpoints/sentry_app_requests.py

```
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases import SentryAppBaseEndpoint, SentryAppStatsPermission
+
+from sentry.utils.sentryappwebhookrequests import SentryAppWebhookRequestsBuffer
+
+from sentry.models import Organization
+
+
+class SentryAppRequestsEndpoint(SentryAppBaseEndpoint):
+    permission_classes = (SentryAppStatsPermission,)
+
+    def format_request(self, request, sentry_app):
+        formatted_request = {
+            "webhookUrl": request.get("webhook_url"),
+            "sentryAppSlug": sentry_app.slug,
+            "eventType": request.get("event_type"),
+            "date": request.get("date"),
+            "responseCode": request.get("response_code"),
+        }
+
+        if "organization_id" in request:
+            try:
+                org = Organization.objects.get_from_cache(id=request["organization_id"])
+                formatted_request["organization"] = {"name": org.name, "slug": org.slug}
+            except Organization.DoesNotExist:
```

@nolachen I think we should add a comment about why we can ignore this error